internal/controller/controllers/agent_reclaimer.go

```diff
@@ -48,16 +51,73 @@ func newAgentReclaimer(hostFSMountDir string) (*agentReclaimer, error) {
 
 func ensureSpokeNamespace(ctx context.Context, c client.Client) error {
 	ns := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: spokeReclaimNamespaceName}}
-	if err := c.Get(ctx, types.NamespacedName{Name: ns.Name}, ns); err != nil {
-		if !k8serrors.IsNotFound(err) {
-			return errors.Wrapf(err, "failed to get namespace %s", spokeReclaimNamespaceName)
-		}
-		if err := c.Create(ctx, ns); err != nil {
-			return errors.Wrapf(err, "failed to create namespace %s", spokeReclaimNamespaceName)
+	mutate := func() error {
+		if ns.Labels == nil {
+			ns.Labels = make(map[string]string)
 		}
+		ns.Labels["pod-security.kubernetes.io/enforce"] = "privileged"
```

Comment:
Just wonder if it worth adding some comment here about why we need to change the security mode. I mean, as it affects the entire namespace it could be confusing if missing the label set here.

Reply:
:shrug: I added this in the commit message:

```
These will ensure that we're able to run privileged pods in this
namespace when Pod Security Standards are adopted in OCP 4.12
```

Also this is not strictly necessary with OpenShift. There should be some label syncer that labels the namespace when we have some SA assigned with privileged permissions, but I think this is the recommended way to go about it.

Reply:
ok, should be good enough I guess to rely on the commit message for reference, or just add this comment about OCP 4.12.